scripts/acceptance

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+cd "${projectRoot}"
+docker-compose down -v
+./scripts/start -d
+
+# run acceptance tests
+cd test/acceptance
+
+  echo "==================================="
+  echo "Running Test using Docker Image"
+  echo "=================================="
+
+  docker-compose down -v
+  docker-compose -f docker-compose.yaml -f docker-compose.override.yaml -f docker-compose.link.yaml up --build --abort-on-container-exit
```

Comment:
Docker compose `run` would probably be more appropriate for running test suites.

Also using the `link` override file will bind a test suite to the end to end payment service network if running locally which may have some unintented side effects.  See comments about changing Docker Compose setup to avoid this issue.